usaspending_api/api_contracts/contracts/v2/search/spending_over_time.md

```diff
@@ -63,6 +71,14 @@ This endpoint returns a list of aggregated award amounts grouped by time period
 + `grant_obligations` (required, number)
 + `direct_payment_obligations` (required, number)
 + `other_obligations` (required, number)
++ `total_outlays` (required, number, nullable)
+    The aggregate outlay amount for this time period and the given filters.
++ `contract_outlays` (required, number)
```

Comment:
Two things:
* Aren't the type-specific values in the format `Contract_Outlays`?
* Should these be nullable?

Reply:
Yes to both of those and the obligation fields should be in the same format. I've updated the API contract to reflect the casing in the JSON response.